python/cudf/cudf/core/column/string.py

```diff
@@ -4297,6 +4297,20 @@ def to_pandas(self, index=None):
             pd_series.index = index
         return pd_series
 
+    @property
+    def values_host(self):
+        """
+        Return a numpy representation of the StringColumn.
+        """
+        return self.to_array()
+
+    @property
+    def values(self):
+        """
+        Return a CuPy representation of the StringColumn.
+        """
+        raise TypeError("Data must be numeric")
```

Comment:
Would similarly raise a `NotImplementedError` here as eventually we'll have a `cudf.arrays.StringArray` similar to Pandas in the future.